mlrun/projects/project.py

```diff
@@ -400,6 +419,11 @@ def _load_project_from_db(url, secrets, user_project=False):
     return db.get_project(project_name)
 
 
+def _delete_project_from_db(project_name, secrets):
+    db = mlrun.db.get_run_db(secrets=secrets)
+    return db.delete_project(project_name)
```

Comment:
If we want to delete project which probably has objects in it, then need to call this with `deletion_strategy=DeletionStrategy.cascade`. Unless this is expected to fail in that case?